Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebKeychainTests/LegacyKeychainTokenStorageTests.swift

```diff
@@ -116,4 +116,53 @@ final class LegacyKeychainTokenStorageTests: XCTestCase {
         keychainStorage.remove()
         XCTAssertEqual(keychainStorage.get(), [])
     }
+    
+    func testSettingLegacyToken() {
```

Comment:
I dont understand why this is bad. KeychainTokenStorage is a class under test, if I set value and then get equal value via this class it should be ok. I think about mocking KeychainStorage in the future to avoid testing on the real keychain.

Reply:
It is not necessarily bad. But it is more of an integration test that tests the combination `LegacyKeychainTokenStorage` and `KeychainTokenStorage` together.

Reply:
Although I see that the rest of the Test class,  tests these classes in conjunction!

Reply:
Yes, this is my plan for the future. I already started it, but wanted to add tests before release.

Reply:
And I have written mock for KeychainStorage. Then I will be able to move this class to other test schema. There is one question more: should we test KeychainStorage itself? its only wrapper for keychain

Reply:
I see what you mean, mostly CRUD stuff called on it. Although it still contatins some logic..
Lets not prioritize it then